build.sh

```diff
@@ -155,7 +159,18 @@ if [ ${BUILD_MACOS} -gt 0 ]; then
 fi
 
 
-mvn ${MAVEN_ARGS} clean install
+mvn ${MAVEN_ARGS} -DskipTests clean install
+
+if [ ${RUN_TEST} -gt 0 ]; then
+    mvn surefire:test@utest
+fi
+
+if [ ${RUN_BENCHMARK} -gt 0 ]; then
+    mvn surefire:test@benchmark
+    if [ ${BUILD_LINUX} -gt 0 ]; then
+       ${BUILDDIR}/linux/JVips/src/test/c/benchmark/SimpleBenchmark ${BASEDIR}/src/test/resources/in_vips.jpg
```

Comment:
Quote paths to avoid issues with spaces.

Reply:
Done